### PR TITLE
feat(root): added missing guidance for section container

### PR DESCRIPTION
**Pull request by GCHQ-Developer-741**
Added guidance for section container that disappeared

## Related issue

#160 

## Checklist

- [x] I have manually accessibility tested any changes, if relevant.

### CODE REVIEW

**Review by GCHQ-Developer-847 — DISMISSED** [review]

File: src/content/structured/components/section-container/guidance.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    { title: "Guidance", path: "/components/section-container" },
+    { title: "Code", path: "/components/section-container/code" },
+  ]
+---
+
+import usageExample from "./images/fig1-container-usage-example.png";
+
+## Introduction
+
+The section container is used by multiple ICDS V2 components to align content more easily.
```
Comment (ad9242) [inline src/content/structured/components/section-container/guidance.mdx:29]:
```suggestion
The section container is used by multiple ICDS components to align content more easily.
```

File: src/content/structured/components/section-container/code.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+---
+
+## Component demo
+
+import { IcSectionContainer } from "@ukic/react";
+
+export const snippetsDefault = [
+  {
+    language: "Web component",
+    snippet: `<ic-section-container>
```
Comment (ad9242) [inline src/content/structured/components/section-container/code.mdx:30]: something weird going on with the spacing of all the snippets: 
image

File: src/content/structured/components/section-container/code.mdx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    <button type="button">End</button>
+  </IcSectionContainer>
+</ComponentPreview>
+
+### Full height
+
+export const snippetsFullHeight = [
+  {
+    language: "Web component",
+    snippet: `<ic-section-container aligned="full-width" fullHeight>
```
Comment (ad9242) [inline src/content/structured/components/section-container/code.mdx:112]:
```suggestion
    snippet: `<ic-section-container aligned="full-width" full-height="true">
```
to be consistent with all other component examples for web-components

**Review by ad9242 — CHANGES_REQUESTED** [review]

**Comment by jd239** [comment]
Worth finding out if there is a better way to demonstrate what section container is doing via the component preview

File: src/content/structured/components/section-container/code.mdx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    <button type="button">End</button>
+  </IcSectionContainer>
+</ComponentPreview>
+
+### Full height
+
+export const snippetsFullHeight = [
+  {
+    language: "Web component",
+    snippet: `<ic-section-container aligned="full-width" fullHeight="true">
```
Comment (ad9242) [inline src/content/structured/components/section-container/code.mdx:112]: sorry, needs to be kebab-case for web-components: 
```suggestion
    snippet: `<ic-section-container aligned="full-width" full-height="true">
```

File: src/content/structured/components/section-container/guidance.mdx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+The components currently using the section container are:
+
+- [Footer](/components/footer)
+- [Hero](/components/hero)
+- [Page header](/components/page-header)
+- [Top navigation](/components/top-nav)
+
+## Usage
+
+When creating a custom component or a web page, a section container can be used to align and group
```
Comment (DBD324) [inline src/content/structured/components/section-container/guidance.mdx:40]: This content does not follow our content style. This whole page needs reviewing.

File: src/content/structured/components/section-container/guidance.mdx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+related content to make the end result easier to understand and digest for the end user.
+
+<DoDontCaution
+  imageSrc={usageExample}
```
Comment (DBD324) [inline src/content/structured/components/section-container/guidance.mdx:44]: This image doesn't follow our image design guidelines and doesn't explain effectively how the section container can be used in different situations.

**Review by DBD324 — CHANGES_REQUESTED** [review]
In general, this page needs to be looked at following the guidance content design guidelines and process. I suggest that this is moved to the design backlog before being merged.

**Pull request closed without merging**